### DISCUSSION TITLE
convert dataframe input to onnxTensor

### DISCUSSION THREAD

**Discussion by ortalbe** [onnx/onnx#4735] [Q&A]
hey 
I took at the following ONNX - Inference on Spark 
https://microsoft.github.io/SynapseML/docs/features/onnx/ONNX%20-%20Inference%20on%20Spark/

the synapse ML support running Onnx model using dataframe as an input.

can we do the same with Scala ? 
I can't found over the web how to convert dataframe to onnxTensor

**Comment by xadupre** [comment]
The converting library onnxmltools is written in python. The model must be loaded in python to work. I'm not aware of a converting library written in scala or java.

**Comment by EmergentOrder** [comment]
For running ONNX models from Scala, I suggest you take a look at [ONNX-Scala](https://github.com/EmergentOrder/onnx-scala). 

Disclaimer: I am the author. So feel free to follow up with any questions or issues in the repo.
